common/core/desktop/src/kmx/kmx_processevent.h

```diff
@@ -138,7 +138,6 @@ const char *Debug_VirtualKey(KMX_WORD vk);
 const char *Debug_UnicodeString(PKMX_WCHAR s, int x = 0);
 const char *Debug_UnicodeString(std::u16string s, int x = 0);
 const char *Debug_ModifierName(KMX_UINT modifiers);
-
```

Comment:
Please restore this file as it is only whitespace change.

Reply:
OK